### DISCUSSION TITLE
How to re-enable Canvas?

### DISCUSSION THREAD

**Discussion by raphm4044** [CosmosOS/Cosmos#2289] [General]
Hi, i disable a Canvas but how to reenable it?

**Comment by AnErrupTion** [comment]
You can use ``canvas.Enable();`` if I'm not wrong.

**Reply by raphm4044** [reply]
wrong

**Reply by quajak** [reply]
It might be worth opening an issue for re-enabling a disabled canvas since the drivers certainly support that.

Currently your only option is to create a new canvas object.
